### PR TITLE
152 fix issues with browser compatability chromium vs firefox

### PR DESCRIPTION
**Pull request by DeltaO3**
I removed scrollbars from (most ) browsers, as they interfere with formatting as positioning. I also figure that the search bar will be used over scrolling anyway. To that end, I have made mild formatting changes to the home page so search is better displayed.
Also included in the PR are some minor bug fixes in main.js (breaking the disconnect alert outside the home page, and adding bs-dark mode on form-control for improved cross browser compat). 

If we want to manually decipher how to customise scrollbars for different browsers we can (e.g. scroll-gutter for chromium browsers but not some android) but it may just be a lot of effort and hard to fully test.

### CODE REVIEW

**Review by livvy10 — APPROVED** [review]
Looks good to me. Tested on edge and chrome.
Do we want the function of the different search bars to be slightly more clear? as in change the placeholders to say "Sign in/log out student by name or ID..." and "Search class list by name or ID...". A bit wordy so not married to this but just to explain which search bar is for what

**Comment by DeltaO3** [comment]
Good idea, will quickly change that
